Exercise/Destroyer/destroyer.js

```diff
@@ -0,0 +1,14 @@
+function destroyer(arr) {
+    var args = arguments;
+    return arr.filter(function(el) {
+        return numToDestroy(args, el);
+    })
+}
+
+function numToDestroy(arr,valueToDestroy){
```

Comment:
Jeśli funkcja zwraca Ci wynik booleanowski to powinna być nazwana w stylu `isInArray`. Takie zmienne albo funkcje zaczynamy wtedy najczęściej od właśnie is, was, has.